Sources/App/QRCodeScanner/Camera/BarcodeScannerCamera.swift

```diff
@@ -0,0 +1,335 @@
+import AVFoundation
+import CoreImage
+import Shared
+import UIKit
+
+class BarcodeScannerCamera: NSObject {
```

Comment:
can you describe what this does and why it's a better solution than e.g. DataScannerViewController?

Reply:
DataScannerViewController is iOS16+